internal/engine/rule_types.go

```diff
@@ -308,7 +308,7 @@ func RuleTypePBFromDB(rt *db.RuleType) (*minderv1.RuleType, error) {
 	var seval minderv1.Severity_Value
 
 	if err := seval.FromString(string(rt.SeverityValue)); err != nil {
-		seval = minderv1.Severity_VALUE_UNKNOWN
+		seval = minderv1.Severity_VALUE_LOW
```

Comment:
I don't think this is an appropriate default for the general severity. It's fine to switch it for GHSA handling, but let's keep the default we already had for minder severity. We don't want to portray `low` risk if the reality is that risk hasn't been triaged yet.